prow/jobs/incubator/compass/components/system-broker/system-broker-generic.yaml

```diff
@@ -0,0 +1,54 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-incubator/compass
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-incubator: "true"
+  extra_refs:
+    - <<: *test_infra_ref
+      base_ref: master
+  branches:
+    - ^master$
+    - ^release-(1\.16)-system-broker$
+  run_if_changed: "^components/system-broker/"
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181204-a6e79be
```

Comment:
Please use image eu.gcr.io/kyma-project/test-infra/bootstrap:v20200831-e46c648b

Reply:
The generic template specifies the `eu.gcr.io/kyma-project/prow/test-infra` repo instead of the `eu.gcr.io/kyma-project/test-infra` - so isn't this the way to go?

https://github.com/kyma-project/test-infra/blob/f6620ce79452db06f520cef9bd1fa7aff0e3c0e5/templates/templates/generic-component.yaml#L59

Reply:
Images in eu.gcr.io/kyma-project/prow/test-infra are not up to date, not maintained anymore and do not contain changes placed during last two years. Newer and maintained versions of bootstrap image are under eu.gcr.io/kyma-project/test-infra. Moreover according to current guidelines repository path eu.gcr.io/kyma-project/prow/test-infra is not valid as it should not contain "prow" part. Please use image which is maintained and located in currently valid repository.

Reply:
@dekiel It's done.